gcsio/src/main/java/com/google/cloud/hadoop/gcsio/CreateFileOptions.java

```diff
@@ -59,7 +71,7 @@ public static Builder builder() {
   public abstract boolean isEnsureNoDirectoryConflict();
 
   /** Whether to overwrite an existing file with the same name. */
-  public abstract boolean isOverwriteExisting();
```

Comment:
wont this break the public interface? Also at other places where we are making changes to Public classes/interfaces

Reply:
It will, but for new major GCS connector release this is allowed.